packages/loader/container-loader/src/container.ts

```diff
@@ -1235,23 +1249,14 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             pkg = quorum.get("code2");
         }
 
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-        return pkg;
-    }
-
-    /**
-     * Loads the runtime factory for the provided package
-     */
-    private async loadRuntimeFactory(pkg: IFluidCodeDetails): Promise<IRuntimeFactory> {
-        const fluidModule = await PerformanceEvent.timedExecAsync(this.logger, { eventName: "CodeLoad" },
-            async () => this.codeLoader.load(pkg),
-        );
-
-        const factory = fluidModule.fluidExport.IRuntimeFactory;
-        if (factory === undefined) {
-            throw new Error(PackageNotFactoryError);
+        if (!isFluidCodeDetails(pkg)) {
+            this.logger.send({
```

Comment:
Should this logging be somewhere else, where we switch packages, not on every read?

Reply:
we only read on package switch/load